resources/js/store/actions.js

```diff
@@ -364,7 +364,17 @@ export const actions = {
     axios
     .post("/media-api/attach", value)
     .then(response => {
-      location.reload()
+      const myEvent = new CustomEvent("customEvent", {
```

Comment:
Would we be able to use a more semantic name for this like `mediaAttachEvent`?
```suggestion
      const attachEvent = new CustomEvent("mediaAttachEvent", {
```

Reply:
100% agree! thank you for catching it